<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inicioDialog</class>
 <widget class="QDialog" name="inicioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Iniciar Sesión</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 255);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="usuarioLabel">
     <property name="text">
      <string>Usuario:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="usuarioLE"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="contrasenaLabel">
     <property name="text">
      <string>Contraseña:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="contrasenaLE">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="iniciarPB">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>Iniciar</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="nuevoUPB">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 255);</string>
     </property>
     <property name="text">
      <string>¿Nuevo?</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="inicioLabel">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Bookin</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="inicio2Label">
     <property name="font">
      <font>
       <family>Arial</family>
      </font>
     </property>
     <property name="text">
      <string>Iniciar Sesión</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
